components/playground/instance/instance.go

```diff
@@ -16,6 +16,7 @@ package instance
 import (
 	"context"
 	"fmt"
+	"github.com/pingcap-incubator/tiup/pkg/localdata"
```

Comment:
Please format the imports.

Reply:
Sorry, I always forget to do this. I've added goimports to `make fmt`, I hope that helps.